### DISCUSSION TITLE
How to delete the comments?

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#8859] [Q&A]
How to delete the comments?
it only can add and edit.

**Comment by o1lab** [comment]
<img width="316" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/9ed9cbfd-1e94-42ca-8b22-e92e545b27c6">

**Reply by c2006878** [reply]
Can you tell me which branch this update is in?

**Reply by dstala** [reply]
0.251.0
